honeybee_radiance_command/getinfo.py

```diff
@@ -30,13 +30,25 @@ class Getinfo(Command):
         * input
     """
 
-    __slots__ = ('_input')
+    __slots__ = ('_input', '_header')
 
-    def __init__(self, options=None, output=None, input=None):
+    def __init__(self, options=None, output=None, input=None, header=None):
         """Initialize Command."""
         Command.__init__(self, output=output)
         self._input = input
         self.options = options
+        self._header = header
+
+    @classmethod
+    def header(cls, options=None, output=None, input=None):
```

Comment:
Similarly I think we should call this `no_header` or something like that to be more explicit.